datasets/hind_encorp/README.md

```diff
@@ -0,0 +1,224 @@
+
+Launchpad.net is a software collaboration platform that hosts many open-source projects and facilitates also collaborative localization of the tools. We downloaded all revisions of all the hosted projects and extracted the localization (.po) files.
+
+Other smaller datasets. This time, we added Wikipedia entities as crawled in 2013 (including any morphological variants of the named entitity that appears on the Hindi variant of the Wikipedia page) and words, word examples and quotes from the Shabdkosh online dictionary.]
+
+### Supported Tasks and Leaderboards
+
+[More Information Needed]
+
+### Languages
+
+[Hindi, English]
+
+## Dataset Structure
+
+### Data Instances
+
+[HindEnCorp Columns
+------------------
+
+The files hindencorp05.plaintext.gz and hindencorp05.export.gz each contain the
+parallel corpus and differ only in the processing of the corpus texts. The
+files have these columns:
+
+- source identifier (where do the segments come from)
+- alignment type (number of English segments - number of Hindi segments)
+- alignment quality, which is one of the following:
+    "manual"  ... for sources that were sentence-aligned manually
+    "implied" ... for sources where one side was constructed by translating
+                  segment by segment
+    float     ... a value somehow reflecting the goodness of the automatic
+                  alignment; not really reliable
+- English segment or segments
+- Hindi segment or segments
+
+Each of the segments field is in the plaintext or export format as described
+above.
+
+If there are more than one segments on a line (e.g. for lines with alignment
+type 2-1 where there are two English segments), then the segments are delimited
+with '<s>' in the text field.]
+
+### Data Fields
+
+[Plaintext vs. Export File Format
+--------------------------------
+
+Both the monolingual and the parallel corpus come in a simple plain text format
+and in a tokenized, tagged and lemmatized format.
+
+The plaintext format preserves the original tokenization (as much as possible
+given the diverse sources included in our corpus).
+
+The 'export' format is tokenized and represents each token as a '|'-delimited
+triple of: the word form, the lemma, and part-of-speech tag. If there was the
+character '|' (this character is also used instead of the proper Devanagari
+Danda in some sources), we escape it as 'pipe;'.
+
+There is exactly the same number of lines in the plaintext and export file
+formats.]
+
+### Data Splits
+
+[More Information Needed]
+
+## Dataset Creation
+
+### Properties of data 
+
+[
+-----------------
+
+All the files are plain text:
+
+- compressed with gzip
+- encoded in UTF-8
+- with unix line breaks (LF)
+- with tab-delimited columns
+
+The monolingual and parallel corpora have different columns.
+
+The actual corpus text is stored in one (monolingual corpus) or two (parallel
+corpus) of the columns.]
```

Comment:
```suggestion
```
Users of the library won't have to deal with the original text files so you can remove this part